### PR TITLE
Update lerna setup

### PR DESCRIPTION
**Pull request by kaufmo**
- fixed versions
- single changelog file

### CODE REVIEW

**Comment by nsams** [comment]
the old pre-2.0 changes should also be consolidated into the new one

File: lerna.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "npmClient": "npm",
     "packages": ["packages/*"],
     "useWorkspaces": true,
-    "version": "independent",
+    "version": "1.3.0",
```
Comment (johnnyomair) [inline lerna.json:5]: Shouldn't this be `2.0.0`?

Comment (kaufmo) [reply lerna.json:5]: no this will be generated on first release with lerna automaticly

**Comment by kaufmo** [comment]
> the old pre-2.0 changes should also be consolidated into the new one

v1 was released independent with separate files I think it would be better when we start with 2.0 clearly in the single CHANGELOG file and the old entries should be inside the packages changelog file.

**Comment by nsams** [comment]
> the old entries should be inside the packages changelog file.

very confusing to have am old changelog file in the packages. At least it should be renamed or deleted.


> EDIT: which order should then the v1 entries be placed? how can the version be combined when they are released independent?
the order in which they are released? (github releases)

**Review by jamesricky — APPROVED** [review]

**Review by chernylu — APPROVED** [review]

**Review by MFSepplive — APPROVED** [review]

**Pull request merged**